### ISSUE TITLE
Release current state

### ISSUE THREAD

**Issue by SPlanzer** [linz/gazetteer#134]
#### Task
Release the plugin to the new plugin repo. This will allow Chris to run some tests if he has time (release often...)

#### Definition of Done
* Once published to new linz plugin repo
* Chris has instructions as to how to connect and consume the plugin

#### Discussion required

Data! he will need a test DB to connect to (@billgeo  fyi)

**Comment by billgeo** [comment]
Chris has mentioned before that he has a test DB he uses (somewhere).

**Comment by billgeo** [comment]
Also, this will be a release to dev plugin repo right?

**Comment by SPlanzer** [comment]
@CMKMS-LINZ I have uploaded an initial release of the plugin to the dev plugin repository (https://plugins.qgis.linz.govt.nz/v1/dev/plugins.xml)

Instructions for connecting to the plugin repository from qgis to download the plugin can be found @ - https://github.com/linz/qgis-plugin-repository#consuming-the-qgis-plugins

I would recommend using QGIS user profiles to separate development plugins and production plugins within you qgis installation. see - https://github.com/linz/qgis-plugin-repository#deployment---production-environment

Note this has been released as the crudest form of a release. I am yet to write any tests. This has however been released under the release often methodology to allow for any initial testing and feedback if you have time to do so. 

I will be back to the project and writing tests next week. This is with the intention to release the first tested version to UAT in one months time.
